src/components/titled-avatar/index.style.js

```diff
@@ -0,0 +1,26 @@
+import styled from 'styled-components';
+import { Link as ReactRouterLink } from 'react-router-dom';
+import { Heading } from '../typography';
+
+export const Container = styled.div`
+  display: flex;
+  align-items: center;
+`;
+
+export const TitleContainer = styled.div`
+  border: 1px solid green;
```

Comment:
we don't have this color in our DS palette's colors

Reply:
Oops. I will fix it 🙈